verilog/analysis/verilog_linter_configuration.h

```diff
@@ -82,7 +82,7 @@ struct RuleBundle {
   // typically that would be a comma or newline.
   bool ParseConfiguration(absl::string_view text, char separator,
                           std::string *error);
-  std::string UnparseConfiguration(char separator) const;
+  std::string UnparseConfiguration(char separator, bool reverse = true) const;
```

Comment:
Why is reverse needed ? Sounds like it was a leftover from some debugging ?

(also, if it is needed, it needs a documentation here).

Reply:
By defauit the output string was coming out with paramters in a reverse alphebetical order, so to avoid breaking things I introduced the reverse boolean so the lint rule print looks better.

FWIW, I commented out the reverse ordered print out and ran all the test cases and it only fails in the 3 tests you'd expect in verilog_linter_configration_test.cc around the UnparseConfiguration. Fixing those and there dosn't seem to be any issues.

I can add some doco, unless you want to do something else here.